thermondo/backend-coding-challenge-ktor
# Add implementation of the challenge

## Description

Add implementation for the challenge

## How to test / reproduce

Run the app and check http://localhost:8080/swagger-ui/index.html
For auth, please login first, then add token from login for locker in requests if needed

## Review checklist

- [+] PR is split into meaningful commits for the ease of reviewing
- [+] Description contains clear instructions on how to test the feature (where applicable)
- [+] Tests have been written
- [-] Appropriate labels have been applied
